AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Why is it only one set, shouldn't it be a distribution?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Ha ha, well I think your confusion is coming from the fact that we're using Bernoulli and not Binomial as the likelihood!</p><p>With Bernoulli, you always ever get 0s and 1s and can only plot their relative frequencies. With Binomial, you do get a distribution of outcomes, and therefore can also convey and plot the uncertainty.</p><p>Actually, thinking about it now, I think we should switch to Binomial from the beginning in this NB:</p><ul><li>It's easier to think about</li><li>It's waaaay more used in practice than Bernoulli, so more practical value for students</li><li>It lets you convey the uncertainty</li><li>It makes Ravin's life easier for the coming lesson</li></ul>

Reply:
Bernoulli or Binomial I don't think affects this. The posterior predictive should give you N *sets* of 0s and 1s, not N 0s and 1s. You compute an average for each one and get N conversion rates, no?

Reply:
<p>Sure. However do we really want to show mean computation over chains and samples here, from inferencedata? Remember this is students first exposure to pymc and arviz</p><p> </p>